packages/i18n/package.json

```diff
@@ -59,7 +59,7 @@
     "ilib": "^14.19.0 || ^14.19.0-webos1"
```

Comment:
can you also update this peer dependency? https://www.npmjs.com/package/ilib-webos?activeTab=versions